modules/admin_manual/pages/installation/deployment_recommendations.adoc

```diff
@@ -234,9 +234,10 @@ management on the application servers.
 ==== Database
 
 MySQL/MariaDB Galera cluster with
-https://mariadb.com/kb/en/mariadb/replication-cluster-multi-master/[master-master replication].
+https://mariadb.com/kb/en/setting-up-replication/[master-slave replication].
```

Comment:
Proposing the following:

**First**
Regarding URL´s as we do this in all the other documents, pls change them **all** to following scheme:

Put the URL you want to show on top into a macro (see examples close below `:toclevels: 1` like
`:setting-up-replication-url: https://mariadb.com/kb/en/setting-up-replication/`

Then use this in the line you want to print it:
`{setting-up-replication-url}[master-slave replication]`
This improves readability and link managability a lot.

**Second**
There is a section `== General Recommendations` recommending `PHP 5.6+`.
Same 2x (!) in `==== Session Management` with a console commands.

This should be changed to the new minimum of PHP 7.3
see
https://doc.owncloud.com/server/10.4/admin_manual/installation/system_requirements.html
and
https://www.php.net/supported-versions.php

**Third**
In section `== References` change the following
from
`https://www.digitalocean.com/community/tutorials/how-to-set-up-a-redis-server-as-a-session-handler-for-php-on-ubuntu-14-04`
to
`https://ubuntuhowtoo.blogspot.com/2019/10/linux-ubuntu-1804-use-redis-for-cache.html`
because our minimum supported Ubuntu release is 18.04...

Reply:
Some URL´s need to be fixed according the suggestion abobe, but this can be done in an other PR. The mentioned once are corrected.